src/Domain/FunctionsOfTime/FixedSpeedCubic.hpp

```diff
@@ -0,0 +1,91 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <array>
+#include <cstddef>
+#include <limits>
+#include <memory>
+#include <pup.h>
+
+#include "DataStructures/DataVector.hpp"
+#include "Domain/FunctionsOfTime/FunctionOfTime.hpp"
+#include "Parallel/CharmPupable.hpp"
+
+namespace domain {
+namespace FunctionsOfTime {
+/* \ingroup ControlSystemGroup
+ * \brief Sets \f$f(t)\f$ and derivatives using cubic rational functions,
+ * such that the first derivative approaches a constant and the second
+ * derivative approaches zero.
+ *
+ * The resultant function of time is
```

Comment:
can you add a blank line before and after the math block to prevent clang-format from messing with it in the future?